dev/com.ibm.ws.org.hibernate.validator.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/LibertyHibernateValidatorExtension.java

```diff
@@ -51,9 +51,14 @@
                         "org.hibernate.validator.cdi.internal.interceptor.ValidationInterceptor;" +
                         "org.hibernate.validator.internal.engine.ValidatorImpl;" +
                         "org.hibernate.validator.internal.engine.ValidatorFactoryImpl;" +
+                        //"org.hibernate.validator.cdi.internal.ValidationProviderHelper;" +
```

Comment:
can undo changes to this file

Reply:
Change due to basing off of feature branch and then switching to master.